### ISSUE TITLE
 How to properly load the dataset

### ISSUE THREAD

**Issue by yue404** [sunyiyou/SAFAS#20]
Hi, I am a beginner, and I want to know how to properly load the dataset. For example, when the protocol is O_C_I_to_M, does the training use all the data from the datasets OCI, or just the train data from them? And does the testing use all the data from the dataset  M or just the test data?

I noticed that in the _’get_single_dataset’_ function within _’datasets‘_, when calling _’FaceDataset’_, the _’label’_ is always None, which means _’video_list'_ contains all the data from a dataset. The _'__getitem__'_ method loads one data sample from the_' video_list',_ and in train.py, when calling _'DataLoader'_, it will iterate over all the data in _'video_list'._ Does this mean that _'train_loader'_ includes all the data from the OCI datasets, implying that all the data from OCI is used for training? If the training should only use the train data from OCI, then I need to modify the _'label'_ value. Thanks a lot.

**Comment by sunyiyou** [comment]
Since we worked on the cross-domain generalization FAS problem, you can use safely all the data points in training domains like O, C and I.
